test/e2e.test.ts

```diff
@@ -19,6 +19,10 @@ describe("SDK test", () => {
   let sdk: Instance;
   let hub: ZNSHub;
 
+  // shkoobyinushnax
+  const tokenOnUniNotCG = "0x000000000427e37c32b2be749610c5e4dd7b6d18";
```

Comment:
Since this is a test file it isn't as big of a deal, but again putting these into a .env or config .json would help reduce code clutter